lib/Send/CopySentMessageHandler.php

```diff
@@ -68,6 +80,11 @@ public function process(Account $account, LocalMessage $localMessage): LocalMess
 			return $localMessage;
 		}
 
+		if(empty($localMessage->getRaw())) {
+			$mail = $this->buildRaw($account, $localMessage);
+			$localMessage->setRaw($mail->getRaw(false));
+		}
```

Comment:
we can probably not fix it but messages where the mime is generated here are not identical to the sent messages. e.g. they won't have a message-id set, so they will not show in threads.